hnn_core/dipole.py

```diff
@@ -123,13 +123,13 @@ def average_dipoles(dpls):
 
     Parameters
     ----------
-    dpls: list of Dipole objects
+    dpls : list of Dipole objects
         Contains list of dipole objects, each with a `data` member containing
         'L2', 'L5' and 'agg' components
 
     Returns
     -------
-    dpl: instance of Dipole
+    dpl : instance of Dipole
```

Comment:
Should this be a "good first issue"? Strict formatting requirements for the documentation :smile:

Reply:
Is there a way to automatically enforce this?